xtas/tasks/_semafor.py

```diff
@@ -92,7 +92,7 @@ def call_semafor(conll_str):
         return Semafor._singleton.call_semafor(conll_str)
 
 
-def to_conll(tree):
```

Comment:
Should this be in _corenlp.py instead? That's what it uses, and the functionality doesn't seem to be specific to this module (although perhaps it's only used here at the moment).